tests/accessibility/settings/therapyScreen.tests.js

```diff
@@ -8,39 +8,42 @@ module.exports = (test) => {
         screen = await settingsScreen.OpenTherapySettings();
     });
     it('has suspend thresold label', async () => {
+        await screen.SwipeUpUntil(screen.SuspendThresholdLabel());
         await expect(screen.SuspendThresholdLabel()).toBeVisible();
     });
     it('has suspend thresold info', async () => {
         await expect(screen.SuspendThresholdInfo()).toBeVisible();
     });
     it('has correction range label', async () => {
+        await screen.SwipeUpUntil(screen.CorrectionRangeLabel());
         await expect(screen.CorrectionRangeLabel()).toBeVisible();
     });
     it('has correction range info', async () => {
         await expect(screen.CorrectionRangeInfo()).toBeVisible();
     });
     it('has pre-meal range label', async () => {
+        await screen.SwipeUpUntil(screen.PreMealRangeLabel());
         await expect(screen.PreMealRangeLabel()).toBeVisible();
     });
     it('has pre-meal range info', async () => {
         await expect(screen.PreMealRangeInfo()).toBeVisible();
     });
     it('has workout range label', async () => {
+        await screen.SwipeUpUntil(screen.WorkoutRangeLabel());
         await expect(screen.WorkoutRangeLabel()).toBeVisible();
     });
     it('has workout range info', async () => {
         await expect(screen.WorkoutRangeInfo()).toBeVisible();
     });
-    it('swipe screen up', async () => {
-        await screen.SwipeUp(screen.BasalRateLabel(), 1);
-    });
     it('has basal rate label', async () => {
+        await screen.SwipeUpUntil(screen.BasalRateLabel());
```

Comment:
[nit] Aren't these two lines a bit redundant. The second line either never be hit if the label is not visible (the first line will throw if it isn't found in maxSwipes) or the second line will always succeed if the label was found by the first line. I suppose it is very clear what you are doing here by having both lines.

Reply:
yeah it is slightly redundant but just felt better with a 'test' also included for clarity. Will probably change my mind at some stage though :)